### DISCUSSION TITLE
Stock or lineageOS boot.img?

### DISCUSSION THREAD

**Discussion by blap** [SebaUbuntu/TWRP-device-tree-generator#73] [Q&A] [answered]
Hi,

I would like to use TWRP in LineageOS, so I need to build tree from lineageOS boot.img or from stock boot.img?
Can I use any of then?

**Comment by SebaUbuntu** [comment, marked as answer]
Doesn't matter, but always use stock one

**Reply by xxroot** [reply]
Both are OK. If the parameters are missing, you can try to use boot and recovery instead
